test/tests/functional/pbs_test_run_count.py

```diff
@@ -112,6 +121,19 @@ def test_less_than_20_run_count(self):
         """
         self.check_run_count(input_count="15", output_count="21")
 
+    def subjob_check(self, jid, sjid, maxruncount="21"):
```

Comment:
Can you add a doc string for functions added?

Reply:
Its a helper function like other existing function which dont have docstrings either

Reply:
Well, this helper function can be reused when we enhance the test suite so docstring is useful. But I'm leaving it up to you.

Reply:
Thanks, the subroutine is small enough to be understood what its doing without doc string. hence i'm passing on this